nextflow.config

```diff
@@ -50,10 +50,26 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
-    //StarAMR options
-    pointfinder_database             = null
-    plasmidfinder_database           = null
-    mlst_scheme                      = null
+    // StarAMR options
+
+    // Databases
+    pointfinder_database             = "Automatic Selection"
+    plasmidfinder_database           = "All"
+    mlst_scheme                      = "Automatic"
+
+    // Additional CLI arguments
+    genome_size_lower_bound              = 4000000
+    genome_size_upper_bound              = 6000000
```

Comment:
I wonder if these should be expanded a bit? For example, we support Campylobacter and it has a genome size of ~1,600,000 base pairs.

Reply:
Also, should this somehow be synced with the nextflow schema values?

Reply:
These are the default values that are set for `staramr search`. Would changing this affect regular `staramr` users? Can you elaborate on what you mean by "synced with the nextflow schema values"

Reply:
I think that's a good suggestion Eric for later. These parameters I believe were originally set for salmonella, with the intention that users change for other organisms.

However, another idea is that, since we are allowing a metadata column containing the species name to be passed to the pipeline, we could use this to automatically set expected genome size ranges. This is something to develop a bit later though, maybe for the next version of the pipeline since the initial request is mainly to port staramr as-is over to a Nextflow pipeline.